includes/class-wc-stripe.php

```diff
@@ -386,6 +387,21 @@ public function install() {
 		}
 	}
 
+	/**
+	 * Redirects to the Stripe settings page upon plugin activation if the transient is set,
+	 * and if not activating multiple plugins at once.
+	 *
+	 * @return void
+	 */
+	public function maybe_redirect_to_stripe_settings(): void {
+		if ( get_transient( 'wc_stripe_redirect_to_settings' ) && ! isset( $_GET['activate-multi'] ) ) {
+			delete_transient( 'wc_stripe_redirect_to_settings' );
+
+			wp_redirect( admin_url( 'admin.php?page=wc-settings&tab=checkout&section=stripe' ) );
```

Comment:
```suggestion
			wp_safe_redirect( admin_url( 'admin.php?page=wc-settings&tab=checkout&section=stripe' ) );
```
It would be better to use `wp_safe_redirect` here.